### DISCUSSION TITLE
Update / Migrate from Typesave-i18n?

### DISCUSSION THREAD

**Discussion by Rar9** [ivanhofer/inlang-plugin-typesafe-i18n#1] [Q&A]
What's the best way to migrate from an existing Sveltekit Typesave-i18n project?

Do you have any recommendations / guide for easy installation?
Is there a easy way to upload translated Typesave-i18n files to inlang?

**Comment by ivanhofer** [comment]
You just need to use this plugin like mentioned in the README to connect your `typesafe-i18n` project with `inlang`.

For now only the basics are implemented in this plugin. If you use `namespaces` or import statements in your translation files, you won't see all messages show up in `inlang`s Editor.
You currently will also need to manually run the `generator` after the Editor makes a commit e.g. as a CI process.
